components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -113,6 +132,24 @@ const ContributorProfileTab = ({
     );
   };
 
+  const emailBody = `Hey ${login}. I'm using OpenSauced to keep track of my contributions and discover new projects. Try connecting your GitHub to https://opensauced.pizza/`;
+
+  const handleInviteClick = () => {
+    const hasSocials = !!(twitter_username || display_email || linkedin_url);
+
+    if (!hasSocials) {
+      handleCopyToClipboard(`${window.location.href}/user/${login}`).then(() => {
```

Comment:
With the changes suggested below, this can be.

```suggestion
      handleCopyToClipboard(`${new URL(`/user/${login}`, location.origin)}`).then(() => {
```